Pyrado/pyrado/domain_randomization/domain_parameter.py

```diff
@@ -202,20 +232,19 @@ def adapt(self, domain_distr_param: str, domain_distr_param_value: Union[float,
 class MultivariateNormalDomainParam(DomainParam):
     """ Domain parameter sampled from a normal distribution """
 
-    def __init__(self, mean: Union[int, float, to.Tensor], cov: Union[list, to.Tensor], **kwargs):
+    def __init__(self, mean: Union[int, float, to.Tensor], cov: to.Tensor, **kwargs):
         """
         Constructor
 
         :param mean: nominal parameter value
         :param cov: covariance
         :param kwargs: forwarded to `DomainParam` constructor
         """
+        assert len(cov.shape) == 2, "Covariance needs to be given as a matrix"
         super().__init__(**kwargs)
 
-        self.mean = to.as_tensor(mean).view(-1)
-        self.cov = to.as_tensor(cov)
-        if not self.cov.ndim == 2:
-            raise pyrado.ShapeErr(msg="The covariance needs to be given as a matrix!")
+        self.mean = to.tensor(mean).view(-1)
+        self.cov = cov
```

Comment:
Warum weicht ihr hier von der Implementierung in simurlacra/master ab?
https://github.com/famura/SimuRLacra/blob/267e22f1c989c123616cdbdd55102239b25f128e/Pyrado/pyrado/domain_randomization/domain_parameter.py#L215

Reply:
Ein Überbleibsel von einem Experiment von uns, wir werden das zurücksetzen. Wir machen derzeit auch noch einige Änderungen an PPO. Wenn wirs abgeändert haben, sagen wir Bescheid. Wegen derzeitigen Klausuren wird das aber noch etwas dauern.